src/main/java/com/rakovets/course/java/core/practice/concurrency/master_and_worker/UserInputException.java

```diff
@@ -0,0 +1,7 @@
+package com.rakovets.course.java.core.practice.concurrency.master_and_worker;
+
+public class UserInputException extends Exception{
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"